tests/unit/source_code/test_notebook_linter.py

```diff
@@ -585,3 +589,51 @@ def test_computes_values_across_notebooks_using_run(extended_test_index, mock_pa
         )
     ]
     assert advices == expected
+
+
+def test_ignores_values_across_notebooks_using_dbutils_notebook_run(extended_test_index, mock_path_lookup):
```

Comment:
can you rewrite this as a functional test? they're easier to read. and maintain. and write as well.

Reply:
done